Formula/clp.rb

```diff
@@ -0,0 +1,125 @@
+class Clp < Formula
+  desc "Linear programming solver"
+  homepage "https://projects.coin-or.org/Clp"
+  url "https://github.com/coin-or/Clp/archive/releases/1.17.3.tar.gz"
+  sha256 "25f0692fe1daa492e7801770af6991506ae9a8c34a4cae358d017400a02dfcf8"
+
+  depends_on "pkg-config" => :build
+  depends_on "gcc"
+  depends_on "openblas"
+
+  resource "coin-or-coin-utils" do
+    url "https://github.com/coin-or/CoinUtils/archive/releases/2.11.2.tar.gz"
+    sha256 "30c7f6c84dbb9f6e4fe5bbe4015ed15e2d1402146f8354cfc50c34d8735a49b1"
+  end
+
+  resource "coin-or-tools-data-netlib" do
+    url "https://github.com/coin-or-tools/Data-Netlib/archive/releases/1.2.6.tar.gz"
+    sha256 "fac8b46bdc4b80eeb321ab437812d3bff1f2887c71822d6a26914ac8bfe3628e"
+  end
+
+  resource "coin-or-tools-data-sample" do
+    url "https://github.com/coin-or-tools/Data-Sample/archive/releases/1.2.11.tar.gz"
+    sha256 "888d21a31e93a529eb3743a92f2ba62b94b3eed4ddc44351feb8034a84c71ec5"
+  end
+
+  def install
+    resource("coin-or-tools-data-netlib").stage do
+      args = %W[
+        --datadir=#{pkgshare}
+        --disable-debug
+        --disable-dependency-tracking
+        --disable-silent-rules
+        --prefix=#{prefix}
+      ]
+      system "./configure", *args
+      system "make"
+      system "make", "install"
+
+      # Remove redundant pkg-config file.
+      rm_f lib/"pkgconfig/coindatanetlib.pc"
+    end
+
+    resource("coin-or-tools-data-sample").stage do
+      args = %W[
+        --datadir=#{pkgshare}
+        --disable-debug
+        --disable-dependency-tracking
+        --disable-silent-rules
+        --prefix=#{prefix}
+      ]
+      system "./configure", *args
+      system "make"
+      system "make", "install"
+
+      # Remove redundant pkg-config file.
+      rm_f lib/"pkgconfig/coindatasample.pc"
+    end
+
+    resource("coin-or-coin-utils").stage do
+      args = [
+        "--datadir=#{pkgshare}",
+        "--disable-debug",
+        "--disable-dependency-tracking",
+        "--disable-silent-rules",
+        "--prefix=#{prefix}",
+        "--includedir=#{include}/clp",
+        "--with-sample-datadir=#{pkgshare}/coin/Data/Sample",
+        "--with-netlib-datadir=#{pkgshare}/coin/Data/Netlib",
+        "--with-blas-incdir=#{Formula["openblas"].opt_include}",
+        "--with-blas-lib=-L#{Formula["openblas"].opt_lib} -lopenblas",
+        "--with-lapack-incdir=#{Formula["openblas"].opt_include}",
+        "--with-lapack-lib=-L#{Formula["openblas"].opt_lib} -lopenblas",
+      ]
+      system "./configure", *args
+      system "make"
+
+      # Deparallelize due to error 1: "mkdir: #{include}/clp/coin: File exists."
```

Comment:
Has this been reported?

Reply:
https://github.com/coin-or/Clp/issues/109